src/open_r1/rewards.py

```diff
@@ -74,6 +74,78 @@ def reasoning_steps_reward(completions, **kwargs):
     return [min(1.0, count / 3) for count in matches]
 
 
+def len_reward(completions, solution, **kwargs):
```

Comment:
Maybe good to start typing our functions to make it easier to understand in/out:

```suggestion
def len_reward(completions : list[Dict[str, str]], solutions : list[str], **kwargs) -> float:
```

Also this should probably be `solutions` instead of the singular `solution` right?